<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSingleWindow</class>
 <widget class="QDialog" name="CSingleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>476</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Long Shot Cyrillic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Single game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
padding: 8px;
color: white;
font: 10pt;
font-family: 'Long Shot Cyrillic'
}

QComboBox {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
}

QPushButton {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
padding: 8px;
color: white;
font: 18pt;
font-family: 'Long Shot Cyrillic'
}

QGroupBox {
	color: white;
	font: 10pt;
	font-family: 'Long Shot Cyrillic';
	border: 2px solid white;
    border-radius: 5px;
}

QLabel {
color: white;
font: 12pt;
font-family: 'Long Shot Cyrillic'
}</string>
  </property>
  <widget class="QLineEdit" name="lineEditNamePlayer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name player</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Bot setting</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of bots</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNumberBots">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLevelBots">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonStartPlay">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start play</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Map</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Size map</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Heigt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMapWidth">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMapHeigt">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>400</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Size planet</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPlanetMinSize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPlanetMaxSize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fly speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFlySpeed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grow speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGrowSpeed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispersion planet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDispersion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Count fleet in planets</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFleetMinCount">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFleetMaxCount">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
